site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -708,4 +708,18 @@ private function sendSocketMessage(array $msg_array): void {
             $this->core->addNoticeMessage("WebSocket Server is down, page won't load dynamically.");
         }
     }
+
+    /**
+     * @Route("/courses/{_semester}/{_course}/queue/student_search", methods={"POST"})
+     */
```

Comment:
```suggestion
    #[Route("/courses/{_semester}/{_course}/queue/student_search", methods: ["POST"])]
```
Switch to PHP8 syntax. See PR #10352